### PR TITLE
[DNM] PGLog Speedups for Jewel

### PR DESCRIPTION
**Pull request by adamemerson**
Backported column family interface to Jewel.

Next up, add support for using column families for the PG Log with Filestore.

### CODE REVIEW

**Comment by yuyuyu101** [comment]
for jewel? we still don't make it on master

**Comment by yuyuyu101** [comment]
and the performance improvement worth to backport?

**Comment by adamemerson** [comment]
I'm in the process of testing to find this out. In master I think the plan is to focus on improving them in Bluestore. For current users of Jewel we need a solution that works in Filestore.

File: src/common/config_opts.h [hunk 1]
```diff
@@ -1134,6 +1134,7 @@ OPTION(filestore_kill_at, OPT_INT, 0)            // inject a failure at the n'th
 OPTION(filestore_inject_stall, OPT_INT, 0)       // artificially stall for N seconds in op queue thread
 OPTION(filestore_fail_eio, OPT_BOOL, true)       // fail/crash on EIO
 OPTION(filestore_debug_verify_split, OPT_BOOL, false)
+OPTION(filestore_pglog_columnfamily, OPT_BOOL, false)
```
Comment (amitkumar50) [inline src/common/config_opts.h]: Have you missed mentioning tracker of backport? Sorry if i missed.
How to verify backport PR?

**Comment by markhpc** [comment]
From what we've seen, column families alone probably is not going to result in a significant speed up, but it's possible that different per-column-family settings might allow us to gain some performance if we can tune for specific behaviors.  The other advantage is that we can more easily track statistics on a per-column-family basis.

**Pull request closed without merging**